### PR TITLE
Introduce events log length config option

### PR DESCRIPTION
**Pull request by fgebhart**
Follow-up PR to #4604.

This adds the config parameter `distributed.scheduler.events-log-length` as a separate config option to accompany the existing `distributed.scheduler.transition-log-length` config option.

Note, that I also changed the default length of both `events-log-length` and `transition-log-length` to `5000` instead of `100000`, since it seems like a more reasonable default lengths to me. Let me know if you have any other suggestion for the default log length value.

~- [ ] Closes #xxxx~
- [x] Tests added / passed
- [x] Passes `black distributed` / `flake8 distributed`

### CODE REVIEW

File: distributed/distributed.yaml [hunk 1]
```diff
@@ -17,7 +17,8 @@ distributed:
     # after they have been removed from the scheduler
     events-cleanup-delay: 1h
     idle-timeout: null      # Shut down after this duration, like "1h" or "30 minutes"
-    transition-log-length: 100000
+    transition-log-length: 5000
```
Comment (mrocklin) [inline distributed/distributed.yaml:20]: If we want to change tihs default value then let's do so in another PR.  I recommend reverting this for now.

File: distributed/config.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "bandwidth": "distributed.scheduler.bandwidth",
     "default-data-size": "distributed.scheduler.default-data-size",
     "transition-log-length": "distributed.scheduler.transition-log-length",
+    "events-log-length": "distributed.scheduler.events-log-length",
```
Comment (mrocklin) [inline distributed/config.py:26]: These aliases are for backwards compatibility.  You don't need to add this here (no one was using the "events-log-length" config value previously.

**Review by mrocklin — COMMENTED** [review]
This seems ok to me.  Two small comments.  Thanks @fgebhart

File: distributed/config.py (hunk 2 above)
Comment (fgebhart) [reply distributed/config.py:26]: good to know, will remove it, thanks!

File: distributed/distributed.yaml (hunk 1 above)
Comment (fgebhart) [reply distributed/distributed.yaml:20]: fine for me 👍

**Comment by fgebhart** [comment]
Removed modifying the default value for the deque lengths.

**Review by jrbourbeau — APPROVED** [review]
Thanks for following up here @fgebhart!
